engine/remote-components/src/main/java/org/datacleaner/descriptors/RemoteDescriptorProvider.java

```diff
@@ -102,25 +110,27 @@ private void downloadDescriptors() {
                             String propertyName = propE.getKey();
                             ComponentList.PropertyInfo propInfo = propE.getValue();
                             String className = propInfo.getClassName();
+                            ConfiguredPropertyDescriptor propDesc;
                             try {
-                                Class cl = Class.forName(className, false, getClass().getClassLoader());
-                                transformer.addPropertyDescriptor(new TypeBasedConfiguredPropertyDescriptorImpl(
+                                Class cl = findClass(className);
+                                transformer.addPropertyDescriptor(propDesc = new TypeBasedConfiguredPropertyDescriptorImpl(
                                         propertyName,
                                         propInfo.getDescription(),
                                         cl,
                                         propInfo.isRequired(),
                                         transformer,
-                                        initAnnotations(component.getName(), propertyName, propInfo.getAnnotations())));
+                                        initAnnotations(component.getName(), propertyName, propInfo.getAnnotations()), propInfo.getDefaultValue()));
                             } catch(Exception e) {
+                                logger.debug("Cannot initialize typed property descriptor '{}'.'{}'", component.getName(), propertyName, e);
                                 // class not available on this server.
-                                transformer.addPropertyDescriptor(new JsonSchemaConfiguredPropertyDescriptorImpl(
+                                transformer.addPropertyDescriptor(propDesc = new JsonSchemaConfiguredPropertyDescriptorImpl(
```

Comment:
Any reason for adding propDesc? It's not used for anything (besides feeding directly into addPropertyDescriptor, of course).